plugin/README.md

```diff
@@ -0,0 +1,28 @@
+# Verifiable Credential Service Worker Plugin for ReSpec
+
+But with support for v2, and no JSON-LD processing.
```

Comment:
What does this "but" compare with?

Reply:
https://github.com/w3c/respec-vc

Reply:
This is not a good sentence —

> https://github.com/w3c/respec-vc but with support for v2, and no JSON-LD processing.

Neither is this —

> **Verifiable Credential Service Worker Plugin for ReSpec** but with support for v2, and no JSON-LD processing.

Nor this —

> **Verifiable Credentials for ReSpec** but with support for v2, and no JSON-LD processing.

Nor this —

> **Verifiable Credential extensions to ReSpec** but with support for v2, and no JSON-LD processing.

Please provide some rephrasing, I guess through a new PR since this one was rushed through.

Reply:
After cycling this through my brain a few more times, I *think* what the above was trying to say is —

```
# Verifiable Credential Service Worker Plugin for ReSpec

The 
**[Verifiable Credential Service Worker Plugin for ReSpec](https://github.com/w3c/vc-jose-cose/tree/main/plugin)** 
is basically the **[Verifiable Credential extensions to ReSpec](https://github.com/w3c/respec-vc)**, 
but with support for VCDM v2, and without JSON-LD processing.
```

Reply:
@TallTed handling this here https://github.com/w3c/vc-jose-cose/pull/241